VTEX - Orders API.json

```diff
@@ -797,14 +797,218 @@
+                                                                                    "description": "Package tracking creation date in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`."
+                                                                                }                                                    
+                                                                            }    
+                                                                        }
+                                                                    }                                      
+                                                                }
+                                                            },
+                                                            "cfop": {
+                                                                "type": "string",
+                                                                "description": "Operations tax code, when applicable. For example, sending promotional gifts to customers.",
+                                                                "nullable": true
+                                                            },
+                                                            "restitutions": {
+                                                                "type": "object",
+                                                                "description": "Information about a value being returned to the customer. This field applies only to invoices with the `Input` type because it relates to returning items.",
+                                                                "properties": {
+                                                                    "Refund": {
+                                                                        "type": "object",
+                                                                        "description": "Customer refund details.",
+                                                                        "properties": {
+                                                                            "value": {
+                                                                                "type": "integer",
+                                                                                "description": "Refund value in cents."
+                                                                            },
+                                                                            "giftCardData": {
+                                                                                "type": "string",
+                                                                                "description": "Refund envolving [Gift Card](https://developers.vtex.com/docs/guides/gift-card-integration-guide).",
```

Comment:
```suggestion
                                                                                "description": "Refund involving [Gift Card](https://developers.vtex.com/docs/guides/gift-card-integration-guide).",
```

Reply:
Which information appears here? ID of the gift card?

Reply:
This field requires way more investigation and will be improved in the future.